CMakeLists.txt

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-cmake_minimum_required(VERSION 3.12)
+cmake_minimum_required(VERSION 3.13)
```

Comment:
Please update the docs with that change.

Reply:
Also you haven't rebuild the Deps in CI and the build still works, so I wonder it that change is needed at all.

Reply:
Docs updated. I did the rebuild+publish earlier in the pipeline that failed because of linter. So the one that passed didn't require deps rebuilding

Reply:
You shouldn't publish until it is ready to be merged because you are messing up with other jobs not related to your commit, and the publish will be overwritten by the nightly build.